openapi/src/parts/controlled_azure_database.yaml

```diff
@@ -0,0 +1,130 @@
+paths:
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/databases:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
```

Comment:
I'm curious to see what this API does if the workspace does not have access to a landing zone. For instance, the storage container API will create a storage account to provision the container in if there is no landing zone to get the storage account from.

If this API errors in the absence of a landing zone I think it should be documented.

Reply:
It also looks like we now require a landing zone for creating a storage container, since the regionality policy work was done.

Reply:
don't all azure things in a workspace require a landing zone at this point?